app/apollo/models/cluster.schema.js

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
We should move all of the model files into /app/models since they will be shared between GraphQL and REST.  Eventually we should use Mongoose in the REST APIs as well (not for this PR).